pages/tag-typy-page.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from "react-helmet";
+import ProjectLoader from '../components/project-loader/project-loader.jsx';
+
+// For pages that display a list of entries based on single param search in the API call
+// e.g., 'tag', 'help_type', ...etc
+class TagTypyPage extends React.Component {
```

Comment:
What does Typy mean? Can we name this after "what it is" (even if that's a slightly longer name?)